joomla/cassiopeia
# Fix colour themes

Pull Request for [https://github.com/joomla/cassiopeia/pull/58](https://github.com/joomla/cassiopeia/pull/58).

### Summary of Changes

- Add new scss files to css build script so that css and css.min will be built.
- Correct typo "autumm" to "autumn" in file name, language string names and value.
- Change "color" (en-US) to "colour" (en-GB) in language string values (but keep en-US language in the variable names because in CSS also "color" is used).

### Testing Instructions

- Code review.
- Test that `npm run build:css` or `npm ci` builds the css and css.min files for the 4 color themes.
- Check that the color themes option still works.
